atd-vze/src/views/Users/Users.js

```diff
@@ -40,7 +43,11 @@ const UserRow = ({ user }) => {
       </td>
       <td>{format(parseISO(user.created_at), "MM/dd/yyyy")}</td>
       <td>{user.logins_count}</td>
-      <td>{format(parseISO(user.last_login), "MM/dd/yyyy")}</td>
+      <td>
+        {user.last_login
```

Comment:
prevents wsod bug when attempting to display users who have never logged in - thanks @mddilley for adding this!

Reply:
Thanks for adding this. I just noticed there is a WSOD when clicking through to the user details page for someone who has never signed in too. Could you add this fix there too? Another hidden bug revealed! 🐛

Reply:
fixed! thanks for catching that @mddilley